File: 1113.cpp
#include <iostream>
using namespace std;
int main(){
    while(true){
        int a, b;
        cin >> a >> b;
        if(a == b){
            break;
        }
        a > b ? cout << "Decrescente\n" : cout << "Crescente\n";
    }
}

File: 1113.java
import java.util.Scanner;
public class Main {
public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        while(true){
                int a = input.nextInt(); int b = input.nextInt();
                if(a == b){
                        break;
                }
                System.out.println((a<b)?"Crescente":"Decrescente");
        }
}
}
